packages/core/README.md

```diff
@@ -1,7 +1,17 @@
 # @contentful/experiences-core
 
-This package provides core internal functionality for the [Experiences SDK](https://www.contentful.com/developers/docs/experiences/set-up-experiences-sdk/).
+### Purpose
+- To contain shared code and utilities across the Experience-builder packages such as constants, types, transformers, and hooks.
+- This means additions to core should be framework agnostic. For example, code that is only compatible with React does not belong in the core package.
 
-## Documentation
+### Concepts
+- **Deep binding**: Relates to the ability to resolve n references. Such an entry containing a reference to another entry which may also contain a reference to another entry and so on.
```

Comment:
"n references" is probably a typo?

Reply:
`n references` is using more mathematical languages, where `n` represents a variable that conveys any number of references. I'll update this to say `ability to resolve multiple references`